### DISCUSSION TITLE
Display Open-Meteo Model

### DISCUSSION THREAD

**Discussion by ghost** [Akylas/oss-weather#117] [Ideas]
Display the Open-Meteo model that is being used for weather in settings.

**Comment by farfromrefug** [comment]
@jacobgonzales20 it is already shown in settings as you can change it. are you asking to see it with "provided by..." ?

**Reply by 0-5-0** [reply]
On the main page I guess, something like.. 

Using Open-Meteo: Best match
Using Open-Meteo: DWD Icon Seamless
Using Open-Meteo: ECMWF IFS 0.4"

or just

Open-Meteo: Best match
Open-Meteo: DWD Icon Seamless
Open-Meteo: ECMWF IFS 0.4"

Not sure how that would look.

**Reply by ghost** [reply]
> @jacobgonzales20 it is already shown in settings as you can change it. are you asking to see it with "provided by..." ?

It would show what model the "Best Match" option is using

**Reply by farfromrefug** [reply]
@jacobgonzales20 not sure I can say what best match is using. the api does not tell me (will check)
but I can show the currently selected one
